inc/saiacl.h

```diff
@@ -277,6 +277,15 @@ typedef enum _sai_acl_action_type_t
     /** Set Forwarding class */
     SAI_ACL_ACTION_TYPE_SET_FORWARDING_CLASS = 0x00000034,
 
+    /** Set ARS monitoring */
+    SAI_ACL_ACTION_TYPE_SET_ARS_MONITORING = 0x00000035,
+
+    /** Set ARS object */
+    SAI_ACL_ACTION_TYPE_SET_ARS_OBJECT = 0x00000036,
```

Comment:
Please clarify how setting the ARS Oid in ACL action relates to a ARS Object (single) bound to NHG object. As we discussed, multiple ARS objects (profiles) can be defined and used e.g. for TCP and non-TCP traffic. Still a single ARS object is on NHG object

Reply:
Route lookup will result in a NHG with ARS object. This is correct. There is 1:1 relationship between NHG and ARS object.

Following two ways can be used to assign an ARS object for a specific flow type.
1. When a flow resolves via NHG with ARS object.
e.g. match<flow-type-tcp> action <SAI_ACL_ACTION_TYPE_DISABLE_ARS_FORWARDING>
In this case TCP flows resolving via NHG with ARS object will go through normal NHG object processing. This rule can be used as a catch all rule for all TCP traffic type, even if NHG do not have any ARS object present and will be noop in this case.

2. When a flow resolves via NHG without ARS object
e.g. match<flow-type-udp> action < SAI_ACL_ACTION_TYPE_SET_ARS_OBJECT>
In this case UDP flows will undergo ARS object processing.
Note: (2) can be used to override ARS object as present in NHG as well. In this case ARS object OID as provided by the ACL rule will be used instead of NHG provided ARS OID.